monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -4479,6 +4561,20 @@ object Observable extends ObservableDeprecatedBuilders {
   def fromIterator[A](resource: Resource[Task, Iterator[A]]): Observable[A] =
     Observable.fromResource(resource).flatMap(fromIteratorUnsafe)
 
+  /** Version of [[fromIterator]] that can work with generic
```

Comment:
I think you need to use `[[fromIterator[A](task* fromIterator(task)]]` or just `fromIterator` to successfully generate doc (currently it fails because of ambiguity)

Reply:
Removed the scaladoc link, considering the other method is just a few lines above.